src/config/configSchema.ts

```diff
@@ -367,6 +367,26 @@ export const configSchema = {
     type: 'string',
     default: '',
   },
+  timestampMarkerEnabled: {
```

Comment:
Not a big deal but I think we can avoid this value by just treating a hotkey value of -1 as disabled. Don't really care either way.